packages/fast-components-react-base/src/viewport-positioner/README.md

```diff
@@ -32,6 +32,8 @@ Inset:
  
  If a default position is specified for a particular axis and there is enough space that position will be set.  Whether there is enough space is determined by comparing the threshold size for the axis (or the height/width of the positioner if no threshold specified) to the available space for that position. If there is not enough space, or if no default position is specified, the positioner chooses the position with the most available space.
 
+ The "verticalLocktoDefault" and "horizontalLocktoDefault" properties force placement to specified default positions regardless of available space.  Note that this has no effect when an axis is in "uncontrolled" mode.
```

Comment:
nit: need to update these to use the capital "T" in "To"

Reply:
yep